doc/config.rst

```diff
@@ -29,7 +29,10 @@ Coverage.py will read settings from other usual configuration files if no other
 configuration file is used.  It will automatically read from "setup.cfg" or
 "tox.ini" if they exist.  In this case, the section names have "coverage:"
 prefixed, so the ``[run]`` options described below will be found in the
-``[coverage:run]`` section of the file.
+``[coverage:run]`` section of the file. If Coverage.py is installed with the
+``toml`` extra (``pip install coverage[toml]``), it will automatically read
```

Comment:
I'd like to leave out the extra.  People using pyproject.toml will already have toml installed.  Can we just say, "If toml is installed, coverage.py will read from "pyproject.toml", etc.

Reply:
I'm not sure toml would necessarily be installed. [poetry] users would have tomlkit installed, [flit] users would have pytoml installed. Users who just used pyproject.toml with setuptools would have no toml libraries installed (pip vendors pytoml). It's your prerogative of course, but personally I'd be happier with `coverage[toml]` in my `dev-requirements.txt` than a separate toml line that I'm not sure where it's used.

Admittedly the text here is a bit clumsy, so it could just say what you suggest instead.

[poetry]: https://github.com/sdispater/poetry
[flit]: https://github.com/takluyver/flit

Reply:
OK, I didn't know there were so many options for TOML parsing. :(  We can keep the extra.